src/Resources/public/contao_files/templates/smartgear/mod_newsreader.html5

```diff
@@ -18,4 +22,22 @@
     </div>
   <?php endif; ?>
 
+
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
```

Comment:
Remove this line. FA will be inserted elsewhere

Reply:
done